portal/account/account.lib.php

```diff
@@ -0,0 +1,269 @@
+<?php
+session_start();
+if ($_SESSION['patient_portal_onsite_two'] && $_SESSION['pid']) {
+    $ignoreAuth_onsite_portal_two = true;
+}
+
+require_once("../../interface/globals.php");
+include_once("$srcdir/patient.inc");
+require_once(dirname(__FILE__) . "/../lib/portal_mail.inc");
+require_once("$srcdir/pnotes.inc");
+
+$action = isset($_REQUEST['action']) ? $_REQUEST['action'] : '';
+
+if ($action == 'set_lang') {
+    $_SESSION['language_choice'] = (int) $_REQUEST['value'];
+    echo 'okay';
```

Comment:
okay

Reply:
Is this debug work or meant to be there?

Reply:
Ajax with a promise should have a response even if not consumed by promise for a purpose. Chose okay.

Reply:
sounds "okay" :)